src/components/swapv2/styleds.tsx

```diff
@@ -7,32 +7,28 @@ import { ButtonEmpty } from 'components/Button'
 import { AutoColumn } from '../Column'
 import { errorFriendly } from 'utils/dmm'
 import { ReactComponent as Alert } from '../../assets/images/alert.svg'
-import Modal from 'components/Modal'
+import Modal, { ModalProps } from 'components/Modal'
+import { Z_INDEXS } from 'styles'
+import { isMobile } from 'react-device-detect'
 
 export const PageWrapper = styled.div`
   display: flex;
   flex-direction: column;
   padding: 24px 16px;
   gap: 24px;
   width: 100%;
-
+  min-height: calc(100vh - 250px); // 100% - header - footer
   @media only screen and (min-width: 768px) {
     flex-direction: column;
-    padding: 24px 16px;
+    padding: 24px 0px 0px 0px;
     gap: 16px;
+    min-height: calc(100vh - 215px); // 100% - header - footer
   }
 
-  // @media only screen and (min-width: 1000px) {
-  //   padding: 24px 32px 100px;
-  // }
-
-  // @media only screen and (min-width: 1366px) {
-  //   padding: 24px 155px 50px;
-  // }
-
-  // @media only screen and (min-width: 1500px) {
-  //   padding: 24px 202px 50px;
-  // }
+  @media only screen and (min-width: 1000px) {
+    ${isMobile ? '' : 'height: calc(100vh - 84px);'} // 100% - header (trigger stịcky form)
+    min-height: '';
```

Comment:
Why set it to '' here?

Reply:
that css line for sticky swap form on PC.  => no need for mobile => ''

Reply:
You can use `unset` bro.

Reply:
his mean the `min-height: '';` line, why has blank string there. Also with the isMobile condition, you can do it like `!isMobile && ...` easier to read I think.

Reply:
yes, i should be 'unset'. got it.